sw/device/tock/boards/opentitan/src/counter_demo.rs

```diff
@@ -4,6 +4,8 @@
 
 //! A simple alarm implementation to display a binary counter over GPIO.
 
+use tock::kernel as kernel;
```

Comment:
You can just write `use tock::kernel;`. This applies everywhere else.

Reply:
Hmmm, I am sure I had issues where in the functions `kernel::<something> `was used, but I will double check - if there are no issues I will change it up.

Reply:
Thank you @mcy , fixing it up now!

Reply:
Done.